sql-statements/sql-statement-alter-user.md

```diff
@@ -91,6 +93,30 @@ ALTER USER 'newuser' ACCOUNT LOCK;
 Query OK, 0 rows affected (0.02 sec)
 ```
 
+```sql
+ALTER USER 'newuser' ATTRIBUTE '{"newAttr": "value", "deprecatedAttr": null}';
+```
+
+```
+Query OK, 0 rows affected (0.02 sec)
+```
```

Comment:
````suggestion
修改 `newuser` 的属性：

```sql
ALTER USER 'newuser' ATTRIBUTE '{"newAttr": "value", "deprecatedAttr": null}';
SELECT * FROM user_attributes;
```

```sql
+-----------+------+---------------------------------------------------+
| USER      | HOST | ATTRIBUTE                                         |
+-----------+------+---------------------------------------------------+
| newuser   | %    | {"newAttr": "value", "deprecatedAttr": null}      |
+-----------+------+---------------------------------------------------+
1 rows in set (0.00 sec)
```
````